### PR TITLE
[ISSUE #119] feat: multi-file log

### PR DESCRIPTION
**Pull request by DHBin**
**Please provide issue(s) of this PR:**
feat #119 

配置项
```yaml
# server启动引导配置
bootstrap:
  # 全局日志
  logger:
    path:
      naming: log/polaris-naming.log
      health-check: log/polaris-health-check.log
      store: log/polaris-store.log
      plugin: log/polaris-plugin.log
      server: log/polaris-server.log
      # 默认日志路径
      default: log/polaris-default.log
```

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [x] Configuration
- [ ] Docs
- [ ] Installation
- [ ] Performance and Scalability
- [ ] Naming
- [ ] HealthCheck
- [ ] Test and Release

**Please check any characteristics that apply to this pull request.**

- [ ] Does not have any user-facing changes. This may include API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: common/log/type.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	StoreLoggerName = "store"
+	// NamingLoggerName 注册日志对象
+	NamingLoggerName = "naming"
+	// PluginLoggerName 插件日志对象
+	PluginLoggerName = "plugin"
+	// ServerLoggerName 接口日志对象
+	ServerLoggerName = "server"
+)
+
+var healthCheckLogger = RegisterScope(HealthCheckLoggerName, "health check logging messages.", 0)
```
Comment (chuntaojun) [inline common/log/type.go:34]: may be

```golang
var (
 //xxx
)
```

will be better

Comment (DHBin) [reply common/log/type.go:34]: I feel the same way

File: common/log/type.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package log
+
+//日志类型
+const (
+	// HealthCheckLoggerName 健康检查日志对象
```
Comment (chuntaojun) [inline common/log/type.go:22]: It’s better to use English instead of related notes.

**Comment by chuntaojun** [comment]
@DHBin  can you fix code conflict ？

**Comment by chuntaojun** [comment]
I think it would be better if you can configure different log levels for different logs

**Review by andrewshan — APPROVED** [review]
LGTM, it could be better when different module log level can be changed separately

**Comment by chuntaojun** [comment]
```yaml
bootstrap:
   logger: 
      naming:
         path: xxxx
         level: debug
```

i think it can be better

**Comment by DHBin** [comment]
```yaml
# server启动引导配置
bootstrap:
  # 全局日志
  logger:
    naming:
      rotateOutputPath: log/polaris-naming.log
      logCaller: true
      # 以下参数不指定或者为零值时，默认值如下
#      rotationMaxSize: 500
#      rotationMaxAge: 30
#      rotationMaxBackups: 100
#      outputLevel: debug
#      stackTraceLevel: none
#      logCaller: false
    health-check:
      rotateOutputPath: log/polaris-health-check.log
      logCaller: true
    store:
      rotateOutputPath: log/polaris-store.log
      logCaller: true
    plugin:
      rotateOutputPath: log/polaris-plugin.log
      logCaller: true
    server:
      rotateOutputPath: log/polaris-server.log
      logCaller: true
    default:
      rotateOutputPath: log/polaris-default.log
      logCaller: true
```

是否考虑当上诉参数不指定或者为零值时，使用默认配置？因为每个参数都指定的话，配置文件显得冗长 @chuntaojun

**Comment by DHBin** [comment]
```yaml
# server启动引导配置
bootstrap:
  # 全局日志
  logger:
    naming:
      rotateOutputPath: log/polaris-naming.log
      logCaller: true
      # 以下参数不指定或者为零值时，默认值如下
#      rotationMaxSize: 500
#      rotationMaxAge: 30
#      rotationMaxBackups: 100
#      outputLevel: debug
#      stackTraceLevel: none
#      logCaller: false
    health-check:
      rotateOutputPath: log/polaris-health-check.log
      logCaller: true
    store:
      rotateOutputPath: log/polaris-store.log
      logCaller: true
    plugin:
      rotateOutputPath: log/polaris-plugin.log
      logCaller: true
    server:
      rotateOutputPath: log/polaris-server.log
      logCaller: true
    default:
      rotateOutputPath: log/polaris-default.log
      logCaller: true
```

是否考虑当上诉参数不指定或者为零值时，使用默认配置？因为每个参数都指定的话，配置文件显得冗长 @chuntaojun

**Comment by chuntaojun** [comment]
> ```yaml
> #      rotationMaxSize: 500
> #      rotationMaxAge: 30
> #      rotationMaxBackups: 100
> #      outputLevel: debug
> #      stackTraceLevel: none
> #      logCaller: false
> ```

日志滚动以及日志等级，这个可以默认值，logCaller这个就没必要参与配置了，默认打开就行

File: common/log/config.go [hunk 3]
```diff
@@ -284,65 +256,76 @@ func processLevels(allScopes map[string]*Scope, arg string, setter func(*Scope,
...
-	// capture gRPC logging
-	if options.LogGrpc {
-		grpclog.SetLogger(zapgrpc.NewLogger(captureLogger.WithOptions(zap.AddCallerSkip(2))))
 	}
-
 	return nil
 }
 
+// defaultOption 设置日志配置的默认值
+func defaultOption(options *Options) {
```
Comment (chuntaojun) [inline common/log/config.go:305]: defaultOption => setDefaultOption

**Review by chuntaojun — APPROVED** [review]

**Review by chuntaojun — APPROVED** [review]

**Pull request merged**